x-pack/plugins/observability_solution/observability_ai_assistant/common/types.ts

```diff
@@ -94,12 +95,12 @@ export interface KnowledgeBaseEntry {
 }
 
 export interface Instruction {
-  doc_id: string;
+  id: string;
   text: string;
 }
 
 export interface AdHocInstruction {
-  doc_id?: string;
+  id?: string;
```

Comment:
`doc_id` can be used by the LLM to lookup entries. I see no reason to expand that concept to instructions. instructions can still have human-readable id's - they do not _have_ to be UUIDs.